edit_here_to_add_blocklists.json

```diff
@@ -1127,5 +1127,50 @@
         "totalDomains": 0,
         "lastUpdated": "",
         "value": 113
+    },
+    {
+        "loc":"custom_filters/normal/id114.txt",
+        "name": "dandelionsprout (anti-malware)",
+        "category": "Malware",
+        "source": "https://raw.githubusercontent.com/DandelionSprout/adfilt/master/Alternate%20versions%20Anti-Malware%20List/AntiMalwareHosts.txt",
+        "totalDomains": 0,
+        "lastUpdated": "",
+        "value": 114
+    },
+    {
+        "loc":"custom_filters/normal/id115.txt",
+        "name": "nextdns (signal)",
+        "category": "Wildcard, Parentalcontrol",
```

Comment:
For wildcard filters, it should be like `"loc":"custom_filters/`**`wildcard`**`/idxxx.txt"`,

Reply:
Yep, my bad. I forgot about that part😂. Il get that fixed asap